src/pkg/services/m365/api/contacts.go

```diff
@@ -81,7 +81,7 @@ func (c Contacts) GetItem(
 	_ *fault.Bus, // no attachments to iterate over, so this goes unused
 ) (serialization.Parsable, *details.ExchangeInfo, error) {
 	options := &users.ItemContactsContactItemRequestBuilderGetRequestConfiguration{
-		Headers: buildPreferHeaders(false, immutableIDs),
+		Headers: newPreferHeaders(idTypeImmutable),
```

Comment:
these lines are not equivalent. The old implementation took a bool (populated as part of the control/options stuff) and decided whether to enable immutable IDs. The new code is always enabling immutable IDs